protocols/0x/index.json

```diff
@@ -0,0 +1,7 @@
+{
+  "cname": "0x",
```

Comment:
Hey sandra, could you please change the cname to 0xgov ? 
It would be better if it's more than three characters